util/dvsim/SimCfg.py

```diff
@@ -659,6 +660,14 @@ def create_bucket_report(buckets):
                     results_str += self.cov_report_deploy.cov_results
                     self.results_summary[
                         "Coverage"] = self.cov_report_deploy.cov_total
+
+                    # Copy cov_report to the report directory and create a
+                    # symlink to the latest directory.
+                    dest_target_dir = os.path.join(self.results_path, "cov_report")
+                    dest_symlink_dir = os.path.join(self.symlink_path, "cov_report")
+                    mk_path(dest_symlink_dir)
+                    shutil.copytree(self.cov_report_dir, dest_target_dir)
+                    mk_symlink(dest_target_dir, dest_symlink_dir)
```

Comment:
IMO it would be better to have `latest` itself be a symlink rather than a separate directory. That way, this won't be necessary. Everytime we create a new `self.results_path`, we can just invoke `clean_odirs` and update the `latest` to point to it. All we would need to do then is just copy over everything we want to publish to `self.results_path`.